egui/src/containers/frame.rs

```diff
@@ -83,7 +120,7 @@ impl Frame {
     /// dark canvas to draw on
     pub fn dark_canvas(style: &Style) -> Self {
         Self {
-            margin: Vec2::new(10.0, 10.0),
+            margin: Vec2::new(10.0, 10.0).into(),
```

Comment:
I think it is better to be explicit in this file
```suggestion
            margin: Margin:symmetric(10.0, 10.0),
```